client.go

```diff
@@ -39,3 +42,17 @@ type Pages struct {
 	Next     *url.Values `json:"next"`
 	Previous *url.Values `json:"previous"`
 }
+
+type ErrorList struct {
+	errors []map[string]string
```

Comment:
Maybe make this field accessible (ie. `Errors` instead of `errors`) or provide an accessor method in case they want to be able to typecast and access each individual error